.github/workflows/_openxla_pin_update_weekly.yml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Code Update PR
+on:
+  schedule:
```

Comment:
Can you also add `workflow_dispatch` so this can be run manually if needed? https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow

Reply:
Thanks, updated